packages/nimble-components/src/patterns/button/styles.ts

```diff
@@ -139,39 +133,42 @@ export const styles = css`
     @layer active {
         .control:active {
             box-shadow: none;
+            color: ${buttonLabelFontColor};
+            border-color: ${borderHoverColor};
             background-image: linear-gradient(
                 ${fillSelectedColor},
                 ${fillSelectedColor}
             );
             background-size: calc(100% - 2px) calc(100% - 2px);
         }
 
+        .control:active [part='start'],
+        .control:active [part='end'] {
+            ${iconColor.cssCustomProperty}: ${buttonLabelFontColor};
+        }
+
         .control:active::before {
             outline: none;
         }
     }
 
     @layer disabled {
         :host([disabled]) {
-            color: ${buttonLabelDisabledFontColor};
             cursor: default;
         }
 
         :host([disabled]) .control {
+            color: ${buttonLabelDisabledFontColor};
```

Comment:
`buttonLabelDisabledFontColor` is the same as `rgba(${actionRgbParitalColor}, 0.3)`, so I replaced it.